roundup.sh

```diff
@@ -258,9 +258,11 @@ do
 
                 # exit subshell with return code of last failing command. This
                 # is needed to see the return code 253 on failed assumptions.
-                # But, only do this if the error handling is activated.
-                set -E
-                trap 'rc=$?; set +x; set -o | grep "errexit.*on" >/dev/null && exit $rc' ERR
+                # But, only do this if the error handling is activated and only if the shell supports it
+                if set -o | grep "^errtrace" >/dev/null; then
```

Comment:
I think this needs to check that errtrace is "on". My output for `set -o` shows "errtrace off" so this line as is will pass even though there is no errtrace. Is that okay?